CLI/Testing/Cxx/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ if(QINPROSTATE001)
     --compare ${QINPROSTATE001}/Baseline/phantom-ktrans.nrrd
     ${TEMP}/${testname}-ktrans.nrrd
     ModuleEntryPoint        
-                --T1Tissue 1597
+                --T1Tissue 1000
```

Comment:
@fedorov Good morning! This was actually an accident. Before I understood the XML file, I was trying to change the GUI default parameters to make testing easier, and mistakenly changed those bits of code. I can submit a quick commit soon reverting those changes back. Thanks for catching that! And it looks like I now know where to look to make my own tests..